pages/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { NextPage } from "next";
 import Head from "next/head";
 import Footer from "../components/footer";
+import MainInfo from "../components/MainInfo";
```

Comment:
Can change to components/footer and components/MainInfo instead of relative path